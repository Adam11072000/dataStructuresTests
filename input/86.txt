init 7117
addImage 3932
deleteImage 3932
deleteImage 6398
addImage 2624
quit