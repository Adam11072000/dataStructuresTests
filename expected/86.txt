init done.
addImage: SUCCESS
deleteImage: SUCCESS
deleteImage: FAILURE
addImage: SUCCESS
quit done.
